lib/icingadb/icingadb-objects.cpp

```diff
@@ -503,6 +505,10 @@ void IcingaDB::UpdateAllConfigObjects()
 		}
 	}
 
+	for (auto& key : globalKeys) {
+		m_Rcon->FireAndForgetQuery({"XADD", "icinga:dump", "*", "type", key, "state", "done"}, Prio::Config);
+	}
+
```

Comment:
Isn’t this one or the below one redundant?

Reply:
Yes, but actually no. Having a signal for every key, no matter if global or type specific, makes it easier to implement in Icinga DB. On the other side, having a key which indicates that everything is done, could be used in Icinga DB Web to indicate the progress of the dump as a whole. We could argue on removing the `*` key, but it's just one query for every dump.